lib/openapi_first/response_validation.rb

```diff
@@ -16,98 +23,8 @@ def call(env)
       return @app.call(env) unless operation
 
       response = @app.call(env)
-      validate(response, operation)
+      Validator.new(operation).validate(response)
```

Comment:
I like that! I suppose (since this is a draft) you are going to have similar approach for the request validation? :)

Reply:
Yes. Very similar. :)